content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -842,6 +842,13 @@ Kubernetes marks a Deployment as _progressing_ when one of the following tasks i
 * The Deployment is scaling down its older ReplicaSet(s).
 * New Pods become ready or available (ready for at least [MinReadySeconds](#min-ready-seconds)).
 
+When the rollout becomes progressing, the Deployment controller adds a DeploymentCondition with the following
+attributes to the Deployment's `.status.conditions`:
+
+* Type=Progressing
+* Status=True
+* Reason=NewReplicaSetCreated | FoundNewReplicaSet | ReplicaSetUpdated
```

Comment:
Although the existing text uses this style, I'd rather not see more of it. How about switching to something like:

```suggestion
* `type: Progressing`
* `status: "true"`
* `reason: NewReplicaSetCreated` | `reason: FoundNewReplicaSet` | `reason: ReplicaSetUpdated`
```

(I think) this is how they'd look in YAML, which is how we usually represent API details. JSON is also OK.

Reply:
Does the Deployment controller also emit an Event? If so, maybe we can mention that?

Reply:
I would rather not in this case. 
1. It emits scaling event. Eg. `Scaled up replica set test to 1`. But this is not reflective of a rollout / deployment and occurs in plain scaling as well.
2. If you create Deployment with 0 replicas there is no event emitted.